mp-migration-guide.md

```diff
@@ -109,50 +109,92 @@ See:
-| **QueryPaymentIds Response**                            |                              |
-| `PaymentIds: [ ]`                                       |                              |
-|                                                         |                              |
-| **QueryPayment**                                        |                              |
-| `paymentid`                                             |                              |
-| `Authorization`                                         |                              |
-| `X-MobilePay-Client-System-Version`                     |                              |
-| `X-MobilePay-Merchant-VAT-Number`                       |                              |
-
-<!-- END_COMMENT -->
+* [ePayment API](https://developer.vippsmobilepay.com/docs/APIs/epayment-api)
+* [ePayment in store](https://developer.vippsmobilepay.com/docs/APIs/epayment-api/how-it-works/vipps-epayment-api-how-it-works-in-store)
+* [Login](https://developer.vippsmobilepay.com/docs/APIs/login-api) - Can be used for [loyalty](https://developer.vippsmobilepay.com/docs/vipps-solutions/loyalty-in-pos)
+* [Order management](https://developer.vippsmobilepay.com/docs/APIs/order-management-api) - Can be used for [receipts](https://developer.vippsmobilepay.com/docs/APIs/order-management-api/vipps-order-management-api#receipts)
+
+### PoS and ePayment endpoints
+
+| Operation                 | MobilePay PoS                            | ePayment                                    |
+|---------------------------|------------------------------------------|---------------------------------------------|
+| PoS management            | `POST/GET/DELETE /v10/pointofsales`      | N/A                                         |
+| Initiate Payment          | `POST /v10/payments`                     | `POST /v1/payments`                         |
+| Initiate Prepared payment | `POST /v10/payments/prepare`             | N/A (For loyalty check [solutions](https://developer.vippsmobilepay.com/docs/vipps-solutions/loyalty-in-pos))     |
+| Query Payment             | `GET /v10/payments/<paymentid>`          | `GET /v1/payments/<paymentid>`              |
+| Query Active Payments     | `GET /v10/payments`                      | N/A                                         |
+| Query payment log         | N/A                                      | `GET /v1/payments/{reference}/events`       |
+| Capture Payment           | `POST /v10/payments/<paymentid>/capture` | `POST /v1/payments/<paymentid>/capture`     |
+| Cancel Payment            | `POST /v10/payments/<paymentid>/cancel`  | `POST /v1/payments/<paymentid>/cancel`      |
+| Refund Payment            | `POST/v10/refunds`                       | `POST /v1/payments/{reference}/refund`      |
+| Lookup a refund           | `GET /v10/refunds/{refundid}`            | `GET /v1/payments/<paymentid>`              |
+
```

Comment:
Nitpick:

There's inconsistent usage of `<id>` and `{id}`. Would it be better to use just one of them?

The ePayment column use both `paymentId` and `reference`, maybe we could use just `reference` to avoid misunderstandings?

Reply:
Apparently I pasted half the MP endpoints in the ePayments column... paymentId is now used only for MP and reference for ePayments as it should. And changed <> to {}

Reply:
It looks like this is resolved now. :-)